discord/member.py

```diff
@@ -171,6 +173,7 @@ def __init__(self, *, data, guild, state):
         self.joined_at = utils.parse_time(data.get('joined_at'))
         self.premium_since = utils.parse_time(data.get('premium_since'))
         self._update_roles(data)
+        self._public_flags = data['user'].get('public_flags', 0)
```

Comment:
If something is inside the `user` payload then surely this belongs within `User` and not `Member`.